getting_started/Data_Input_Perf.md

```diff
@@ -0,0 +1,24 @@
+## Performance of Data Input Pipeline
+Overview of supported data input pipelines: https://github.com/google/maxtext/blob/main/getting_started/Data_Input_Pipeline.md
+
+### HuggingFace pipeline
+The following data are collected using c4 data in Parquet format.
+| Pipeline    | seq_len | per_host_batch | # of host | # of batch | first step (s) | total time (s) |
+| ----------- | ------- | -------------- | --------- | ---------- | -------------  | -------------- |
+| HuggingFace | 2048    | 8              | 1         | 1000       | 6              | 52             |
+| HuggingFace | 2048    | 8              | 64        | 1000       | 5              | 49             |
+| HuggingFace | 2048    | 32             | 1         | 1000       | 7              | 191            |
+
+### Grain pipeline
+The following data are collected using c4 data in ArrayRecord format.
+| Pipeline    | seq_len | per_host_batch | # of host | # of batch | worker | first step (s) | total time (s) |
+| ----------- | ------- | -------------- | --------- | ---------- | -----  | -------------- | -------------- |
+| Grain       | 2048    | 8              | 1         | 1000       | 1      | 7              | 479            |
+| Grain       | 2048    | 8              | 64        | 1000       | 1      | 6              | 354            |
+
+### TFDS pipeline
+The following data are collected using c4 data in TFRecord format.
+| Pipeline    | seq_len | per_host_batch | # of host | # of batch | first step (s) | total time (s) |
+| ----------- | ------- | -------------- | --------- | ---------- | -------------  | -------------- |
+| HuggingFace | 2048    | 8              | 1         | 1000       | 28             | 141            |
```

Comment:
Should the Pipeline here by TFDS instead of HuggingFace?

Reply:
Thanks for catching this. Fixed.